infrastructure/quick-deploy/aws/all/terraform.tfvars

```diff
@@ -0,0 +1,303 @@
+  "entity"           = ""
+  "bu"               = ""
+  "owner"            = ""
+  "application code" = ""
+  "project code"     = ""
+  "cost center"      = ""
+  "Support Contact"  = ""
+  "origin"           = "terraform"
+  "unit of measure"  = ""
+  "epic"             = ""
+  "functional block" = ""
+  "hostname"         = ""
+  "interruptible"    = ""
+  "tostop"           = ""
+  "tostart"          = ""
+  "branch"           = ""
+  "gridserver"       = ""
+  "it division"      = ""
+  "Confidentiality"  = ""
+  "csp"              = "aws"
+  "grafanaserver"    = ""
+  "Terraform"        = "true"
+  "DST_Update"       = ""
+}
+
+
+vpc = {
+  enable_private_subnet = false
+}
+
+# AWS EKS
+eks = {
+  cluster_version = "1.22"
+  cluster_autoscaler = {
+    node_selector = { "grid/type" = "Operator" }
+  }
+  cluster_endpoint_public_access = true
+  docker_images = {
+    cluster_autoscaler = {
+      image = "k8s.gcr.io/autoscaling/cluster-autoscaler"
+      tag   = "v1.23.0"
+    }
+    instance_refresh = {
+      image = "public.ecr.aws/aws-ec2/aws-node-termination-handler"
+      tag   = "v1.15.0"
+    }
+  }
+}
+
+# Operational node groups for EKS
+eks_operational_worker_groups = [
+  {
+    name                                     = "operational-worker-ondemand"
+    spot_allocation_strategy                 = "capacity-optimized"
+    override_instance_types                  = ["c5.xlarge"]
+    spot_instance_pools                      = 0
+    asg_min_size                             = 1
+    asg_max_size                             = 5
+    asg_desired_capacity                     = 1
+    on_demand_base_capacity                  = 1
+    on_demand_percentage_above_base_capacity = 100
+    kubelet_extra_args                       = "--node-labels=grid/type=Operator --register-with-taints=grid/type=Operator:NoSchedule"
+  }
+]
+
+# EKS worker groups
+eks_worker_groups = [
+  {
+    name                                     = "worker-c5.4xlarge-spot"
+    spot_allocation_strategy                 = "capacity-optimized"
+    override_instance_types                  = ["c5.4xlarge"]
+    spot_instance_pools                      = 0
+    asg_min_size                             = 0
+    asg_max_size                             = 1000
+    asg_desired_capacity                     = 0
+    on_demand_base_capacity                  = 0
+    on_demand_percentage_above_base_capacity = 0
+  }
+]
+metrics_server = {
+  node_selector = { "grid/type" = "Operator" }
+}
+keda = {
+  node_selector = { "grid/type" = "Operator" }
+}
+elasticache = {
+  engine             = "redis"
+  engine_version     = "6.x"
+  node_type          = "cache.r4.large"
+  num_cache_clusters = 2
+}
+mq = {
+  engine_type        = "ActiveMQ"
+  engine_version     = "5.16.4"
+  host_instance_type = "mq.m5.xlarge"
+}
+mongodb = {
+  image_name    = "mongo"
+  image_tag     = "5.0.9"
+  node_selector = { "grid/type" = "Operator" }
+  #persistent_volume = {
```

Comment:
this comment can be removed? I see that the persistent volume is considered in the corresponding module

Reply:
The comment is here to show an example of how you can defined a PV.